data/json/items/generic.json

```
@@ -1621,6 +1621,26 @@
     "to_hit": -2,
     "material": [ "steel", "plastic" ]
   },
+  {
+    "id": "retool_300blk",
```

This needs to be more specific. See #23078.
```suggestion
    "id": "retool_ar15_300blk",
    "type": "GUNMOD",
    "name": ".300 AAC Blackout conversion kit",
    "description": "A barrel, upper receiver, and all the other parts required to convert an AR-15 for .300 AAC Blackout.",
    "weight": 1000,
    "volume": 2,
    "integral_volume": 0,
    "integral_weight": 0,
    "price": 50000,
    "material": "steel",
    "symbol": ":",
    "color": "green",
    "location": "bore",
    "mod_targets": [ "ar15" ],
    "install_time": 60000,
    "ammo_modifier": "300blk",
    "min_skills": [ [ "weapon", 3 ], [ "mechanics", 2 ] ]
```